roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -176,11 +176,15 @@
     openshift_logging_kibana_nodeselector: "{{ openshift_logging_kibana_ops_nodeselector }}"
     openshift_logging_kibana_cpu_limit: "{{ openshift_logging_kibana_ops_cpu_limit }}"
     openshift_logging_kibana_memory_limit: "{{ openshift_logging_kibana_ops_memory_limit }}"
+    openshift_logging_kibana_cpu_request: "{{ openshift_logging_kibana_ops_cpu_request }}"
+    openshift_logging_kibana_memory_request: "{{ openshift_logging_kibana_ops_memory_request }}"
     openshift_logging_kibana_hostname: "{{ openshift_logging_kibana_ops_hostname }}"
     openshift_logging_kibana_replicas: "{{ openshift_logging_kibana_ops_replica_count }}"
     openshift_logging_kibana_proxy_debug: "{{ openshift_logging_kibana_ops_proxy_debug }}"
     openshift_logging_kibana_proxy_cpu_limit: "{{ openshift_logging_kibana_ops_proxy_cpu_limit }}"
     openshift_logging_kibana_proxy_memory_limit: "{{ openshift_logging_kibana_ops_proxy_memory_limit }}"
+    openshift_logging_kibana_proxy_cpu_request: "{{ openshift_logging_kibana_ops_proxy_cpu_request }}"
+    openshift_logging_kibana_proxy_memory_request: "{{ openshift_logging_kibana_ops_proxy_memory_request }}"
```

Comment:
Yes, as I mentioned in my comment above. The variables in openshift_logging_elasticsearch have been renamed and removed from the include_role tasks. I didn't if someone is currently working on this. I could add the necessary parameters again, if you want me to.